### DISCUSSION TITLE
How to use wingetcreate for create portable program new manifest?

### DISCUSSION THREAD

**Discussion by soroshsabz** [microsoft/winget-create#560] [Q&A]
I want to add manifest for maven and I want to doing that with wingetcreate

but I see below error

```
Downloading and parsing: https://dlcdn.apache.org/maven/maven-3/3.9.9/binaries/apache-maven-3.9.9-bin.zip...
No supported installer(s) found in zip archive.
```

**Comment by soroshsabz** [comment]
related to https://github.com/microsoft/winget-pkgs/pull/189469

**Comment by mdanish-kh** [comment]
The package does not contain a WinGet supported installer type. See [supported installer formats](https://learn.microsoft.com/en-us/windows/package-manager/winget/#supported-installer-formats). Also as the [review comment](https://github.com/microsoft/winget-pkgs/pull/189469#discussion_r1832995682) pointed out, scripts are disallowed by policy
